docroot/modules/features/resource/config/optional/views.view.mirador_view.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
     - core.entity_view_mode.node.mirador
     - node.type.resource
     - user.role.administrator
+    - user.role.anonymous
```

Comment:
Change the view access to permission based. Users having access to view published content should be able to access mirador view.